code-style/README.md

```diff
@@ -5,6 +5,8 @@
 * **[Rails](rails)**
 * **[Pattern Libraries](pattern_libraries)**
 * **[Markdown](markdown)**
+* **[PHP](php)**
+* **[JavaScript](javascript)**
```

Comment:
Oh, I forgot to add the JS link when adding that section. Nice catch, thanks for adding this! 👏